starlark/testdata/bytes.star

```diff
@@ -0,0 +1,131 @@
+# Tests of 'bytes' (immutable byte strings).
+
+load("assert.star", "assert")
+
+# bytes(string) -- UTF-k to UTF-8 transcoding with U+FFFD replacement
```

Comment:
Add a case with an unpaired surrogate. And a case with a proper surrogate pair. If `bytes(string)` doesn't support unpaired surrogates, then construct the bytes object with a literal and inspect that it has the expected contents with indexing.